src/herbpy/wam.py

```diff
@@ -223,10 +230,10 @@ def MoveUntilTouch(manipulator, direction, distance, max_distance=None,
         @param max_distance maximum distance in meters
         @param max_force maximum force in Newtons
         @param max_torque maximum torque in Newton-Meters
-        @param ignore_collisions collisions with these objects are ignored when 
+        @param ignore_collisions collisions with these objects are ignored when
```

Comment:
I think this name is misleading. Is this "continue colliding with this object(e.g. table) until max_force, max_torque are reached"? If so, MoveUntilTouch is dependent on actually colliding with this object, but the name seems to suggest that it's not relevant whether this object is touched or not. Unless my interpretation is mistaken, we should change this to something more like `collide_with`. If I am mistaken, this parameter needs clearer documentation.